pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1062,17 +1062,6 @@ public void openLedgerFailed(ManagedLedgerException exception, Object ctx) {
             managedLedgerConfig.setRetentionSizeInMB(retentionPolicies.getRetentionSizeInMB());
             managedLedgerConfig.setAutoSkipNonRecoverableData(serviceConfig.isAutoSkipNonRecoverableData());
             OffloadPolicies offloadPolicies = policies.map(p -> p.offload_policies).orElse(null);
-
-            if (offloadPolicies == null) {
```

Comment:
This code was there to fallback to defaults in case there is no special offload policy for a given namespace. How is this behavior maintained?

Reply:
It's handled by https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java#L813

Reply:
Got it. Thanks!